### PR TITLE
Update special-parameters.adoc

### PR DESCRIPTION
**Pull request by giordanolucas**

### CODE REVIEW

File: modules/ROOT/pages/special-parameters.adoc [hunk 1]
```diff
@@ -93,6 +93,8 @@ When a `ParameterResolver<Type>` is received to resolve the expression value, th
 
 WARNING: Expression resolution is deferred until it is required, so `resolve()` can fail due to an invalid expression.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/special-parameters.adoc]:
```suggestion
[WARNING]
====
* Expression resolution is deferred until it is required, so `resolve()` can fail due to an invalid expression.
* Parameter resolvers are allowed only in operations. As such, you cannot use parameter resolvers in configurations.
====
```

File: modules/ROOT/pages/special-parameters.adoc [hunk 2]
```diff
@@ -93,6 +93,8 @@ When a `ParameterResolver<Type>` is received to resolve the expression value, th
...
 
+WARNING: Parameter Resolvers are only allowed in Operations. Dont try to use these in Configurations.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/special-parameters.adoc]:
```suggestion
```

**Review by sarathecoubian — CHANGES_REQUESTED** [review]
Warning block. If these should not be grouped together, please let me know.

**Comment by giordanolucas** [comment]
@sarathecoubian Just added the suggested changes

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**